ophyd/controls/cas/motor.py

```diff
@@ -35,6 +35,21 @@
 
 
 class CasMotor(CasRecord):
+    '''A fake EPICS motor record, made available to EPICS by the built-in
+    channel access server.
+
+    Parameters
+    ----------
+    name : str
+        The record name (not including the server prefix)
+    positioner : Positioner
+        The ophyd :class:`Positioner` to expose to EPICS
+    tweak_value : float
+        The default tweak value
+    **kwargs : dict
```

Comment:
Or, rather it typically goes in the description like "kwargs are passed through to base-class, see CasRecord for possible keys"